lib/ssl/test/ssl_cipher_SUITE.erl

```diff
@@ -74,9 +74,14 @@ end_per_group(_GroupName, Config) ->
 
 init_per_testcase(_TestCase, Config) ->
     ct:timetrap({seconds, 5}),
-    Config.
+    put(log_level, debug),  %% SSL log level
```

Comment:
Maybe I am missing something but is this verifying that the debug-level macro works by manual inspection of the logs?

Reply:
shouldn't we check log content explicitly in one test case?